cypress/integration/userCanSaveRecipeToCookbook.spec.js

```diff
@@ -0,0 +1,25 @@
+describe('Save recipe to Cookbook', () => {
+  beforeEach(() => {
+    cy.route({
+      method: 'GET',
+      url: 'http://localhost:3000/v1/recipes/1',
+      response: 'fixture:single_recipe.json',
+      status: 200
+    })
+
+    cy.route({
+      method: 'POST',
+      url: 'http://localhost:3000/v1/recipes/1/favorite',
+      response: '{"message": "The recipe was successfully added to your favorites"}',
+      status: 201
+    })
+  })
+  
+  it('Successfully saves recipe to the Cookbook list', () => {
+    cy.anotherLoginUser('user2@mail.com', 'password')
```

Comment:
Cannot the creator of the recipe add his/her own recipe to his/her cookbook? Since you're logging in as another user?

Reply:
In this feature user would save to their cookbook only recipes created by other users :)

Reply:
Should that condition not be reflected in the title of the story?